index.html

```diff
@@ -1,14 +0,0 @@
-<!DOCTYPE HTML>
-<html lang="en-US">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="1; url=https://nilearn.github.io/stable/index.html">
-        <script type="text/javascript">
-            window.location.href = "https://nilearn.github.io/stable/index.html"
```

Comment:
This file is required as it implements the redirection to `stable/index.html`.

Reply:
Ah ok I see, I added it back.

Reply:
And what is `index_save.html`? Is that also relevant?

Reply:
It's a copy of the automatically generated `index.html` (now in `dev` and `stable` folders). I'm not sure why it was kept, but we don't need it anymore.